do-not-commit/avoid-pyyaml-load.py

```diff
@@ -0,0 +1,13 @@
+import yaml
+
+
+# ruleid:avoid-pyyaml-load
+yaml.load("!!python/object/new:os.system [echo EXPLOIT!]")
```

Comment:
Avoid using `load()`. `PyYAML.load` can create arbitrary Python
objects. A malicious actor could exploit this to run arbitrary
code. Use `safe_load()` instead. ([serialization.avoid-pyyaml-load.avoid-pyyaml-load](https://semgrep.dev/r/serialization.avoid-pyyaml-load.avoid-pyyaml-load))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=serialization.avoid-pyyaml-load.avoid-pyyaml-load) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>